### PR TITLE
#5488 divider fix

### LINKED ISSUE #5488: divider in collection activity
image
image
- should not be full width divider

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5488
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-04-03 at 7 45 06 PM

### CODE REVIEW

**Review by daiagi — CHANGES_REQUESTED** [review]
the hover color in ```moreNFTDetails``` needs to extend to the edge of container

image

File: components/collection/activity/ownerInsightsTabs/FlipperTab.vue [hunk 1]
```diff
@@ -65,7 +65,7 @@
           <NFTsDetaislDropdown :flips="flips" variant="Flippers" />
         </div>
       </div>
-      <hr class="my-3" />
+      <hr class="m-3" />
```
Comment (daiagi) [inline components/collection/activity/ownerInsightsTabs/FlipperTab.vue:68]:
```suggestion
      <hr class="my-3 mx-5" />
```

**Review by daiagi — CHANGES_REQUESTED** [review]
the hr margin should be same in flippers and in holders tab

**Review by daiagi — APPROVED** [review]
lgtm

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 33.45 USD/KSM ~ 0.299 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xe3a5853f9a61af3971ab3e64bd542f069cf00b1a71a7b07045e2b87639913c25](https://kusama.subscan.io/extrinsic/0xe3a5853f9a61af3971ab3e64bd542f069cf00b1a71a7b07045e2b87639913c25)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**